database/migrations/2022_10_24_002838_update_logs_table.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('logs', function (Blueprint $table) {
+            $table->dropForeign(['mensa_id']);
+            $table->dropColumn('mensa_id');
+            // 0       Emergency: system is unusable
```

Comment:
Things to think about, are there certain logs which we don't want to expose to barco/bestuur? Do we now have a way to filter those out?

Reply:
not yet. I have not yet had the possibility to test backend to frontend communication. Need to use postman I guess